app/containers/ItunesContainer/index.js

```diff
@@ -40,56 +88,116 @@ export function ItunesContainer({ dispatchItuneTracks, dispatchClearItuneTracks,
 
   const debouncedHandleOnChange = debounce(handleOnChange, 200);
 
+  const renderTrackList = () => {
+    const results = get(tracksData, 'results', []);
+    const resultCount = get(tracksData, 'resultCount', 0);
+    return (
+      <If condition={!isEmpty(results) || loading}>
+        <Skeleton loading={loading} active>
+          <If condition={!isEmpty(trackName)}>
+            <div>
+              <T id="search_query_text" values={{ trackName }} />
+            </div>
+          </If>
+          <If condition={resultCount !== 0}>
+            <div>
+              <T id="matching_tracks" values={{ resultCount }} />
+            </div>
+          </If>
+          <Grid>
```

Comment:
we can remove this. We already set `Grid` as parentComponent in `For` component

Reply:
removed it